docs/release_notes/2025/3-4-4.md

```diff
@@ -0,0 +1,58 @@
+---
+id: 3-4-4
+description: Release notes for version 3.4.4
+title: Prophecy 3.4.4.x
+tags:
+  - release notes
+  - changelog
+  - january
+---
+
+January 31, 2025
+
+- Prophecy Python libs version:
+- Prophecy Scala libs version:
+
+## Features
+
+### Expanded business rules capabilities
+
+The new [business rule](docs/Spark/functions/business-rules-engine/business-rules-engine.md) capabilities include:
+
+- Setting a timeframe in which business rules should apply.
+- Automatic detection of conflicting rule logic.
+- Defining multiple output columns per business rule.
+
+### Enhanced Data Explorer experience
+
+The [Data Explorer](docs/Spark/execution/data-explorer.md) has been enhanced in the following ways:
+
+- Filter and sort happens in the frontend. It no longer requires execution in the backend.
+- You can search your data using the new **Search** function.
+- You can load larger data samples at a time using the new **Selective** data sampling setting.
+
+## Updates
+
+### New custom gem function
+
+When you are [building a custom Spark gem](docs/extensibility/gem-builder/gem-builder-reference.md), you can include the `customOutputSchemaEnabled` function in your gem code to enable or disable to **Custom Schema** toggle by default. This toggle can be found in the **Output** tab of a gem.
+
+### Advanced mode for target model
```

Comment:
on quick glance folks might not realize this is for SQL, so I suggest adding "SQL" in the heading or description